nimbus.fml.yaml

```diff
@@ -131,17 +131,20 @@ types:
     HomeScreenSection:
       description: The identifiers for the sections of the homescreen.
       variants:
-        top-sites:
+ # These values were updated to camelCase in order to match the currently
+ # running experiment. Once that experiment is over in v100, these values
+ # should be returned to kebab-case.
```

Comment:
Would be nice to add a ticket to change it back to kebab-case for v100. If we already have a ticket for that then you can ignore this comment @electricRGB :)

Reply:
[Documented](https://mozilla-hub.atlassian.net/browse/FXIOS-3997). :)